packages/nextjs/app/builders/0x6BF7d6b94282BD48ff458599aDafA268BcB009FF/page.tsx

```diff
@@ -0,0 +1,147 @@
+              {profile.name}
+            </h1>
+            <h2 className="text-lg text-gray-600 dark:text-gray-400 mb-3 transition-colors duration-300">
+              {profile.title}
+            </h2>
+
+            {/* Location */}
+            <div className="flex items-center text-gray-600 dark:text-gray-400 mb-4 transition-colors duration-300">
+              <StyledIcon icon={FaMapMarkerAlt} size={16} className="mr-2 text-gray-600 dark:text-gray-400" />
+              <span className="text-sm">{profile.location}</span>
+            </div>
+
+            {/* Bio */}
+            <p className="text-gray-600 dark:text-gray-300 text-center mb-6 px-4 text-sm leading-relaxed transition-colors duration-300">
+              {profile.bio}
+            </p>
+
+            {/* Social Links */}
+            <div className="flex space-x-6 mb-6">
+              {/* GitHub */}
+              <a
+                href={profile.social.github}
+                target="_blank"
+                rel="noopener noreferrer"
+                className="text-gray-600 dark:text-gray-400 hover:text-gray-900 dark:hover:text-gray-200 transition-colors duration-300 ease-in-out"
+              >
+                <StyledIcon
+                  icon={FaGithub}
+                  size={24}
+                  className="text-gray-600 dark:text-gray-400 hover:text-gray-900 dark:hover:text-gray-200"
+                />
+              </a>
+
+              {/* Twitter */}
+              <a
+                href={profile.social.twitter}
+                target="_blank"
+                rel="noopener noreferrer"
+                className="text-gray-600 dark:text-gray-400 hover:text-blue-400 transition-colors duration-300 ease-in-out"
+              >
+                <StyledIcon
+                  icon={FaTwitter}
+                  size={24}
+                  className="text-gray-600 dark:text-gray-400 hover:text-blue-400"
+                />
+              </a>
+
+              {/* LinkedIn */}
+              <a
+                href={profile.social.linkedin}
+                target="_blank"
+                rel="noopener noreferrer"
+                className="text-gray-600 dark:text-gray-400 hover:text-blue-600 transition-colors duration-300 ease-in-out"
+              >
+                <StyledIcon
+                  icon={FaLinkedin}
+                  size={24}
+                  className="text-gray-600 dark:text-gray-400 hover:text-blue-600"
+                />
+              </a>
+
+              {/* Email */}
+              <a
+                href={`mailto:${profile.email}`}
+                className="text-gray-600 dark:text-gray-400 hover:text-red-500 transition-colors duration-300 ease-in-out"
+              >
+                <StyledIcon
+                  icon={FaEnvelope}
+                  size={24}
+                  className="text-gray-600 dark:text-gray-400 hover:text-red-500"
+                />
```

Comment:
This code seems a bit repetitive, perhaps you could use an array and map over it?